tests/plugins/toolsets/grafana/test_grafana_loki.py

```
@@ -1,57 +1,80 @@
 import os
+from typing import Any
+from holmes.core.tools import ToolResultStatus, ToolsetStatusEnum
+from holmes.plugins.toolsets.grafana.grafana_api import grafana_health_check
 import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)